extension/chrome/elements/compose-modules/compose-draft-module.ts

```diff
@@ -141,6 +142,7 @@ export class ComposeDraftModule extends ViewModule<ComposeView> {
         } else {
           await this.doUploadDraftWithLocalStorageFallback(mimeMsg, msgData, async () => {
             await this.view.emailProvider.draftUpdate(this.view.draftId, mimeMsg, this.view.threadId);
+            await this.localDraftRemove(); // delete local draft if there is one
```

Comment:
have you considered moving this call to doUploadDraftWithLocalStorageFallback()?
Also, msgData parameter seems unused and can be refactored out

Reply:
Done, thanks for noticing 👍